fink_science/ztf/hostless_detection/processor.py

```diff
@@ -18,7 +18,7 @@
 import os
 
 import numpy as np
-from pyspark.sql.functions import pandas_udf
+from pyspark.sql.functions import col, pandas_udf, sum
```

Comment:
As `sum` is a built-in function in Python, I would rather use:

```
import pyspark.sql.functions as F
```

and use `F.col`, `F.sum` and `F.pandas_udf` in the code to avoid potential future mismatches by redefining `sum`.